react/src/components/ModelServiceSettingModal.tsx

```diff
@@ -93,6 +93,7 @@ const ModelServiceSettingModal: React.FC<Props> = ({
         loading: mutationToUpdateService.isLoading,
       }}
       title={t("modelService.EditModelService")}
+      centered
```

Comment:
Let's create a type of BAIModal and use it instead of adding centered to the ant.d Modal. It has centered as a default and its own CSS for overriding, which does not apply to others.